sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateMutableProjection.scala

```diff
@@ -124,6 +124,7 @@ object GenerateMutableProjection extends CodeGenerator[Seq[Expression], MutableP
 
         public java.lang.Object apply(java.lang.Object _i) {
           InternalRow ${ctx.INPUT_ROW} = (InternalRow) _i;
+          /*** project list: ${expressions.map(_.toCommentSafeString).mkString(", ")} */
```

Comment:
No problem.
But, it is directly related with the intention of original @rxin 's PR, too.
I think we need to make some consensus on how to improve that.
I'll wait and collect other people's opinions until this afternoon, and do that.